### DISCUSSION TITLE
Let use `alert()` in titletemplate

### DISCUSSION THREAD

**Discussion by shirobachi** [TriliumNext/Trilium#3538] [Ideas]
What you guy think about let this label to work:
```
#titleTemplate="${alert('Title')} [${alert('Year')}]" 
```

**Comment by zadam** [comment]
The template is evaluated on the backend so alert would not do anything there ...

**Reply by shirobachi** [reply]
Oh, that is sad, but thanks a lot for info. Still learning trillium
